recipes/tinygrad/recipe.yaml

```diff
@@ -0,0 +1,137 @@
+context:
+  version: "0.10.1"
+  build_number: 0
+  # see github.com/conda-forge/conda-forge.github.io/issues/1059 for naming discussion
+  torch_proc_type: ${{ "cuda" ~ cuda_compiler_version | version_to_buildstring if cuda_compiler_version != "None" else "cpu" }}
+  tests_to_skip: >
+    _not_a_real_test
+    ${{ "fails `assert Device.DEFAULT in failed_platforms`" if 0 }}
```

Comment:
First time seeing this type of comments. Surprised it works o.o
Makes me wonder if we never discussed `{# #}` equivalents for the new recipe format.

Reply:
> Makes me wonder if we never discussed `{# #}` equivalents for the new recipe format.

Yeah, I really don't like the way comments work in this context, but this is what our rattler translation crew came up with in https://github.com/conda-forge/torchvision-feedstock/blob/main/recipe/recipe.yaml, so I didn't explore this further.

CC @wolfv